recipes/configure.rb

```diff
@@ -31,6 +71,8 @@
   recursive true
 end
 
+node.override['nexus_iq_server']['logging']['archived_file_count'] = 5
```

Comment:
I don't think this override belongs here.

Reply:
Removed this (2cdb85f).